cloud-init/gophish-dir-setup.tpl.yml

```diff
@@ -0,0 +1,21 @@
+---
+runcmd:
+  # Create a subdirectory on the EFS share for GoPhish data
+  - [mkdir, -p, "${efs_gophish_data_dir}"]
```

Comment:
It's be nice if this link was easy to find for the users.  😉

Reply:
Ha, that is not the link that I was thinking of in this case.  I want to give them a desktop shortcut to the local GoPhish web page.  I know the `.desktop` file will look something like this:
```
[Desktop Entry]
Version=1.0
Type=Link
Name=GoPhish - Login
Icon=user-bookmarks
URL=https://localhost:3333/login?next=%2F
```

The tricky part is putting that file in the VNC user's home directory, since we look up the VNC user's username in SSM.  I was hoping to find a system-wide location where I could put that file so that I don't have to look up the VNC user's username just for this.  I tried to do something similar to [what you did here](https://github.com/cisagov/ansible-role-xfce-cool/blob/develop/tasks/main.yml#L27-L32), but no matter what I tried, the shortcut wouldn't display on the VNC user's desktop.

Reply:
I'm going to create the desktop shortcut in [`pca-gophish-composition-packer`](https://github.com/cisagov/pca-gophish-composition-packer) where it will be simpler and cleaner.